src/components/Menu/MenuItem/MenuToggle.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { useState } from 'react';
+import {ReactComponent as CloseIcon} from "assets/icon-close.svg";
+import './MenuItem.scss';
+
+type MenuToggleProps = {
+    children: React.ReactNode;
+    direction: 'left' | 'right';
+    index: number;
+    onToggle?: (active: boolean) => void;
+    toggleStartLabel: string;
+    toggleStopLabel: string;
+}
+
+function MenuToggle(props: MenuToggleProps) {
+
+    const [status, setStatus] = useState(false);
+
+    return (<button 
+                className={`menu-item menu-item${props.index} menu-item${status ? '--active' : '--disabled'} menu-item--${props.direction}`}
```

Comment:
This looks complicated. Try to use https://www.npmjs.com/package/classnames instead and implement a rule such as `classNames('menu-item', 'menu-item${props.index}', { 'menu-item--active': status, 'menu-item--disabled': !status }, ...)`

Reply:
Done ✅